src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    public Circle(String name, int side, String color) {
+        super(name, side, color);
+    }
+
+    @Override
+    public void getsSquare() {
+        System.out.println("Figure: " + getName()
+                + "side " + getSide()
+                + " Area " + (Math.PI * getSide() * getSide())
+                + " Color: " + getColor());
+    }
```

Comment:
You merged 2 different things.
1 - getArea() that should return some decimal value
2 - draw() that should print info regarding figure (and may reuse getArea() method of course)

Please introduce second interface and split logic between 2 different methods